test/integration/video/nockFixtures/SigningKeys/del.json

```diff
@@ -0,0 +1,83 @@
+[
+    {
+        "scope": "https://api.mux.com:443",
+        "method": "post",
+        "path": "/video/v1/signing-keys",
+        "body": {},
+        "status": 201,
+        "response": {
+            "data": {
+                "private_key": "LS0tLS1CRUdJTiBSU0EgUFJJVkFURSBLRVktLS0tLQpNSUlFcEFJQkFBS0NBUUVBd2RxUndnTTB3SWUvbGt0cS8yRDlYblR6VXFCYk8rdW9FTU4vcjlOdTByS0lsdjByCk1IWk1RQTZKYk1LWHZ4cHhHNmxDYklpN2RucmJXYWZLU0MrZ0lmRFM4QjdUNEYvUXF0cy9HSE9lM3pkODgvL1AKdlJTMHdUUUJtUDFNbmRBbEh5dnRQTW9EQ3ErMlBJWm9LQkRnZDkzTWkzWURFWjIwSngwdXQxK3VLa3FnM1VOSgo3QWswaHRpYVdqckJCYUZzbEluc01IdVZjYVlqcGsvb2FwR0pzVC9BUnYvRzNWZ0NDdnBRUUNSVHZCdHkyMTNPCi9GQjJaTXlhd0xxTGZUdG1LZ0N6S3VTREROSjBwMVpCZ0xHNWJJRjd0MzVFelFGUmpLaWgwZEY5VzhOaWZzYTgKY28xcWdwNGZhQWYrTUh4L1lTUHVCdGJnVEJSS2p1YW1JbXkwY3dJREFRQUJBb0lCQUVIWExaaGFwR2tiOGh0bQo1Q0YyM0xkZk8yRjk2MTdQckZaOVAvb0U4dWdIRVZOUEM0TTdPcXlyQTB2NFQ3ZnRrQ3NYOHVLUUVqT3NpS0hMCjFtMXNGNVJQaWlWYUl4WmJGTEZqSSt2M0lTZXBYZncxajJZbUNUcHhvblZHMnVYaTB5amlMNEp0RGxscjBwWisKQkJSUG54MjNMSWpWZlRkWGU5K3BEcnZqZUhFT0xaNThyL3BRWmZWL3hVam5Qb1NzSkhHazU3M2ZRVTlqZmphLwp6eEpKajdSTkNPN0ZPZ3RDYkNoK2ZzR1dGMXFWcGlLVGdhWlNaSFErUThhV25wVDF1UWp4Slg5RjJWRFR5QWU2CnpCWjRrSlcxWEQ1R0ZYL3lpOVh5Nk5RWDJSUTZsZ29XV1EvYjlTQ1NhL0lZTmxVaHlYZUxOZCtncE5LV0RnZHYKdjA2RDlOa0NnWUVBK0J0U3ZkSkphRytyMEgyRXZPbWZvZVBQSy9zUVFqcVRtbmZyZGlRdFViVncrcWxyN1V6VQp6UWloTmRxTUV2eWxVMmQwc1F0VU9iYzRCc0RFTTJpZjRHQnJrbTRlZ1YzT09VNGk0d0gvM1JFOVVHaldOUzZqCmpQUzQwaEtoTlIzdHY4NG54UlowbkZwMW94WTM1bE83ZENHVE1ueHpDNUs4RHNnRS9zVU4zajBDZ1lFQXlBVmoKcDdQd05wRlJmWmRzVUNHNkZQZE9rWHVaSVoyVk1RQWluZTU0aDR5RUR4anlVaWgxT29UalZKRmJrQ0djVWgrWgo1VTBTczNQbEQ4bFVBWWR3NzBGUkp2RnVzaXR4d2RXNVBoSVNkOTZTelVTZnRzTUU4czlvcW8xOE1yWDZPbjZ6Cnl2Ty9sNnV6SzNrdHlXTWtPS3hlUXZSdk5qUWJmV1ZQVVlTOE9HOENnWUVBcmtRQ3lmRjZQakN4cmE5OEhIUzcKK3luWHlqeCtId2FseGttVXlWWi92blNDOW8wcGtYUDF4RTNiYmNnanV2Slo3VXN0VzIxcHEyQlZxZWliNXNBRwpDOE90eXh3dHNSQUNXWjNZUXpxVVRPSmUvcXVrU1phSXFlajZLQk9OeDVGTi9Qc0FWQjZCcVpDZ0wvemJKcHlKCmZmaFhPNDdOLzhNQTlDRHA0VWpuaE1FQ2dZRUFzdFBtcWd2OUNyeFhrQ0t1YXVJTWZTd3lLVzNJbGI1T29YV08KZllhMGFmUGVLYVZUR05NbEpBR2xRWG1IMUUvSWM3L2ZGcHdXeVN3bnNqcDhpYlZ1NmVtS3hYZDdVWDM0QWFXMApMV040OFlTajJUT3VXM2IyV0tQSTN4c0ozRWpJaEQyOVB3azMxVUZrT2N6b2U3SFFzMzZmODlxWUIyMEUydEtWCjNMc0FrSzhDZ1lBWHNhTDNyZ2EwTjJYU0YvZkRWM3Z5U05xdXQ3d1FiTlFxNjZ5VlpGMG1WL0xqamFad2dENjUKay9DQ05hZnhSRXBXR3UzTmxFL29ydmpyM2NxY2pjUlNhSm4rbk1mb01kT1FEN0FFOStrNEgweHFvRy9VbWhuNQphVU1qQzA3Wk9zWnFOWE5IWHlTRWFqV2FBR0VaZVo0alBzeXprQkRCUlJRREtsaTJBczZrY1E9PQotLS0tLUVORCBSU0EgUFJJVkFURSBLRVktLS0tLQo=",
+                "id": "D02mhgvCmh00S3HcueIFQwqZZGubyX4wb6",
+                "created_at": "1545180554"
+            }
```

Comment:
Is this key associated with a test account?

Reply:
Yeah good catch. It's with a test account with no assets, so I don't think there's much for someone to sign for, but we should definitely make sure to be careful if we decide to make other nocks.